[IMP] eCommerce: update of cross-selling doc for Odoo 15.0 and after (#1712)
odoo/documentation: content/applications/websites/ecommerce/maximizing_revenue/cross_selling.rst

================================================
Cross-selling: Accessories and Optional Products

> ================================================

Reviewer: Why did you mark it as solved?
Isn't it named "accessory products"?
(and not "accessory" alone)

> When selling products, it is natural to look for new ways to stimulate customers to buy more goods,

Reviewer 2: Any sales process is an opportunity to stimulate up-selling. For example, with an extra-warranty or some accessories.

> When selling products, it is natural to look for new ways to stimulate customers to buy more goods,

Reviewer 2: ```suggestion
Any sales process is an opportunity to stimulate up-selling. For example, with an extra-warranty or some accessories.
```

> some accessories.

Author: Indeed, they are **not** the same. I suppose I wasn't focused enough when editing the text and lost the most important info... My bad :sweat_smile:

> such as an extra-warranty or some accessories.

Reviewer 2: ```

To do so, you can use *Cross-selling*, a sales technique consisting in selling a customer something
in addition to the product or service they were originally shopping for. It is a great way to
maximize the value of each one of your customers.

It can be done via two different features:

    - Accessory products on the checkout page.
    - Optional products on a new *Add to Cart* screen.

.. _cross_selling/accessory:

Accessory Products
==================

*Accessory Products* are products that are displayed when customers review their cart before paying.
As an example, you could suggest a mouse or a keyboard when someone buys a computer.

> To add *Accessory Products*, go the :menuselection:`Website --> Products --> Products` and, under

Reviewer 2: Website --> Products --> Products --> select a product and, under....

the **Sales Tab**, add your accessories to the **Accessory Products** field.

    .. image:: cross_selling/accessory-product.png
       :align: center
       :alt: Add an accessory product to product form.

Now, go to your website and add the product to your cart. On the payment step, you can see your
accessory products suggested to the customer.

>     .. image:: cross_selling/accessory-products-customer-view.gif

Reviewer: Indeed @Lau-Odoo  I meant gif ^^

       :align: center
       :alt: See of payment view and accessory product suggestion.

.. _cross_selling/optional:

Optional Products
=================

*Optional products* are products directly related to an item. Each time the product is added to a
cart, a pop-up appears, suggesting other products to buy.

To enable optional products, go to :menuselection:`Website --> Configuration --> Settings` and tick
*Optional Products*. Then, go to :menuselection:`Website --> Products --> Products` and open the
item you want to add an optional product to. In the **Sales tab** of that product, add as many

> optional products as you want.

Reviewer: I know "item" is best to avoid repeating the same words, but since we're talking about views in Odoo, I'm wondering if it wouldn't be best to write "product".

    .. image:: cross_selling/optional-product.png
       :align: center
       :alt: Add an optional product to product form.

> Now, go to your website and add the product to your cart. A pop-up appears, suggesting you to

Reviewer 2: ```suggestion
From now, when a customer adds this product to their cart, a pop-up appears and suggests to also add the optional product(s).
```

> buy the optional product(s) you have added to the list.

Reviewer 2: ```suggestion

```

>     .. image:: cross_selling/cross-selling-pop-up.gif

Reviewer: same comment

================================================
Cross-selling: Accessories and Optional Products
================================================

> Any sales process is an opportunity to maximize revenues. For example, with an extra-warranty

Reviewer: "extra warranty" (no hyphen) or maybe "extended warranty" that is more accurate.

or some accessories.

To do so, you can use **Cross-selling**, a sales technique consisting in selling a customer something
in addition to the product or service they were originally shopping for. It is a great way to
maximize the value of each one of your customers.

It can be done via two different features:

- :ref:`Accessory products <cross_selling/accessory>` on the checkout page.
- :ref:`Optional products <cross_selling/optional>` on a new :guilabel:`Add to Cart` screen.

.. _cross_selling/accessory:

Accessory Products
==================

**Accessory Products** are products that are displayed when customers review their cart before paying.
As an example, you could suggest a mouse or a keyboard when someone buys a computer.

> To add *Accessory Products*, go the :menuselection:`Website --> Products --> Products` and, under

Reviewer: ```suggestion
To add *Accessory Products*, go to :menuselection:`Website --> Products --> Products` and, under
```

the :guilabel:`Sales Tab`, add your products to the :guilabel:`Accessory Products` field.

    .. image:: cross_selling/accessory-product.png
       :align: center
       :alt: Add an accessory product to product form.

Now, go to your website and add the product to your cart. On the payment step, you can see your
accessory products suggested to the customer.

    .. image:: cross_selling/accessory-products-customer-view.png
       :align: center
       :alt: See of payment view and accessory product suggestion.

.. seealso::

>    `Alternative, Accessory, and Optional Products

Reviewer: ```suggestion
   `Odoo Tutorials: Alternative, Accessory, and Optional Products
```

   <https://www.odoo.com/slides/slide/alternative-accessory-and-optional-products-637>`_.

.. _cross_selling/optional:

Optional Products
=================

**Optional products** are products directly related to an item. Each time the product is added to a
cart, a pop-up appears, suggesting other products to buy.

To enable optional products, go to :menuselection:`Website --> Configuration --> Settings` and tick
:guilabel:`Optional Products`. Then, go to :menuselection:`Website --> Products --> Products` and
open the item you want to add an optional product to. In the :guilabel:`Sales tab` of that product,
add as many optional products as you want.

    .. image:: cross_selling/optional-product.png
       :align: center

>        :alt: Add an optional product to product form.

Reviewer 3: Don't indent `image` directives for no reason. It could lead to display glitches if we add new content right above it.

Author: My bad :)

> From now, when a customer adds this product to their cart, a pop-up appears and suggests to also add
> the optional product(s).

Reviewer: ```suggestion
From now, when a customer adds this product to their cart, a pop-up appears and suggests also adding
the optional product(s).
```

>     .. image:: cross_selling/cross-selling-pop-up.png

Reviewer: "cross-selling" is already in the folder's name. Not a mistake per se, but it can be named "pop-up" imo.
